test/Date/xplatInterval.js

```diff
@@ -8,13 +8,20 @@ if (WScript.Platform && WScript.Platform.OS != "win32") {
     function rand() {
         return parseInt(Math.random() * 1e2) + 50;
     }
+
     for (var j = 0; j < 1e2; j++) {
         var pre_time = Date.now(), now;
         for(var i = 0; i < 1e6; i++) {
             now = Date.now();
             var diff = now - pre_time
 
-            if (diff < 0) throw new Error ("Timer interval has failed. diff < 0");
+            // INTERVAL_FOR_TICK_BACKUP = 5
+            // So, anything beyond 5ms is not subject to our testing here.
```

Comment:
> in case system time jumps back <= 5 ms, we most likely recover that internally. (due to our cache is 5ms)

How does it recover? Suppose last time it returned `cacheSysTime + 2` using cache. Now 3 ticks later, system time jumps to `cacheSysTime - 1`. It'll return from line 89 and fail the test (now - prev == -3).

(Just try to understand this test stability.)